NEWS.md

```diff
@@ -6,6 +6,10 @@
 
 ## Changes to defaults
 
+* `duplicate_argument_linter()` no longer produces warnings for functions that allow 
```

Comment:
I would reword the news item to more closely match what's done -- `mutate()` and `transmute()` are added the default of `exclude=`

Reply:
Okay, I have changed this item. Now it reads:

```md
* `duplicate_argument_linter()` defaults to `except = c("mutate", "transmute")`.
  This is because these functions allow sequential updates to variables (#1345, @IndrajeetPatil).
```